### PR TITLE
DocBlock fixes

### PR DESCRIPTION
**Pull request by donatj**
This PR corrects a small handful of documentation errors. 

This is an initial dipping of the toes. 

There is a fair bit more I feel can be greatly improved, and I'd be happy to do so if the community is open to it.

- [X] Is this related to documentation?

### CODE REVIEW

File: src/Reader/Extension/AbstractFeed.php [hunk 1]
```diff
@@ -115,12 +115,12 @@ public function toArray() // untested
         return $this->data;
     }
 
-    /**
-     * Set the XPath query
-     *
-     * @param  DOMXPath $xpath
-     * @return AbstractEntry
-     */
+	/**
```
Comment (froschdesign) [inline src/Reader/Extension/AbstractFeed.php]: Please revert the changes for indentation.

File: src/Reader/Extension/Atom/Feed.php [hunk 2]
```diff
@@ -476,10 +476,13 @@ protected function getAuthorFromElement(DOMElement $element)
         return $author;
     }
 
-    /**
-     *  Attempt to absolutise the URI, i.e. if a relative URI apply the
-     *  xml:base value as a prefix to turn into an absolute URI.
-     */
+	/**
```
Comment (froschdesign) [inline src/Reader/Extension/Atom/Feed.php]: Same here.

File: src/Reader/FeedSet.php [hunk 3]
```diff
@@ -129,9 +129,12 @@ private function resolveRelativeUri($link, $scheme, $host, $uriPath)
         );
     }
 
-    /**
-     *  Canonicalize relative path
-     */
+	/**
```
Comment (froschdesign) [inline src/Reader/FeedSet.php]: Same here.

File: src/Writer/Feed.php [hunk 4]
```diff
@@ -112,6 +112,7 @@ public function removeEntry($index)
      * added to a feed container in order to be indexed.
      *
      * @param int $index
+	 * @return Entry
```
Comment (froschdesign) [inline src/Writer/Feed.php]: Same here.

File: src/Writer/Renderer/Entry/Atom.php [hunk 5]
```diff
@@ -373,9 +373,12 @@ protected function _setContent(DOMDocument $dom, DOMElement $root)
         $root->appendChild($element);
     }
 
-    /**
-     * Load a HTML string and attempt to normalise to XML
-     */
+	/**
```
Comment (froschdesign) [inline src/Writer/Renderer/Entry/Atom.php]: Same here.

File: src/Writer/Renderer/Entry/AtomDeleted.php [hunk 6]
```diff
@@ -29,11 +29,11 @@ public function __construct(Writer\Deleted $container)
         parent::__construct($container);
     }
 
-    /**
-     * Render atom entry
-     *
-     * @return \Zend\Feed\Writer\Renderer\Entry\Atom
-     */
+	/**
```
Comment (froschdesign) [inline src/Writer/Renderer/Entry/AtomDeleted.php]: Same here.

File: src/Writer/Renderer/Feed/AtomSource.php [hunk 7]
```diff
@@ -28,11 +28,11 @@ public function __construct(Writer\Source $container)
         parent::__construct($container);
     }
 
-    /**
-     * Render Atom Feed Metadata (Source element)
-     *
-     * @return \Zend\Feed\Writer\Renderer\Feed\Atom
-     */
+	/**
```
Comment (froschdesign) [inline src/Writer/Renderer/Feed/AtomSource.php]: Same here.

**Review by froschdesign — CHANGES_REQUESTED** [review]

File: src/Reader/Extension/AbstractFeed.php (hunk 1 above)
Comment (donatj) [reply src/Reader/Extension/AbstractFeed.php]: Weird. I don't know how that happened, my editor should have been set to spaces.  I'll clean these up.

**Comment by donatj** [comment]
@froschdesign Rebased based on your notes. 

Sorry about that, I'm usually pretty careful to match indentation styles of projects, so I don't know how that happened.

**Comment by weierophinney** [comment]
Thanks, @donatj!
